valued.gemspec

```diff
@@ -17,10 +17,12 @@ Gem::Specification.new do |spec|
 
   spec.files =
     Dir.chdir(File.expand_path('..', __FILE__)) do
-      `git ls-files -z`.split("\x0").reject do |f|
-        f.match(%r{^(test|spec|features)/})
-      end
+    `git ls-files -z`.split("\x0").reject do |f|
+      f.match(%r{^(test|spec|features)/})
     end
+  end
+  spec.files += `git ls-files -- spec/valued/shared_examples/*.rb`.split("\n")
+
```

Comment:
Shouldn't this be included in the above? Why do we need to add this explicitly?

Reply:
You are right :facepalm: Removed.